src/reconcile/rec_write.c

```diff
@@ -6591,3 +6594,37 @@ __rec_dictionary_lookup(
 	*dpp = next;
 	return (0);
 }
+
+/*
+ * __verbose_lookaside_write --
+ *	Create a verbose message to display with details about the transaction
+ * state when performing a lookaside table write.
```

Comment:
Can you elaborate on what is wrong with that indentation? I'm not certain what I'm missing.